library/src/test/java/com/whatsapp/stringpacks/StringPacksTest.java

```diff
@@ -32,17 +33,20 @@
 public class StringPacksTest {
 
   @NonNull private final Locale zhLocale = new Locale("zh");
+  @NonNull private final Locale zhTWLocale = new Locale("zh", "TW");
+  @NonNull private final Locale haNGLocale = new Locale("ha", "NG");
   @NonNull private final Locale enLocale = new Locale("en", "US");
 
   @Mock Resources resources;
   @Mock Configuration configuration;
   @Mock AssetManager assetManager;
+  @Mock StringPacksLocaleMetaDataProvider stringPacksLocaleMetaDataProvider;
 
   private Application application;
 
   @Before
   public void setUp() {
-    MockitoAnnotations.initMocks(this);
+    MockitoAnnotations.openMocks(this);
```

Comment:
Why this change?

Reply:
initMocks is depricated and the replacement is openMocks

Reply:
This change is not a required for this PR. Please revert this one and if needed raise another PR for this change